client/platforms/windows/daemon/windowsfirewall.cpp

```diff
@@ -484,57 +549,6 @@ bool WindowsFirewall::allowTrafficTo(const QHostAddress& targetIP, uint port,
   return true;
 }
 
-bool WindowsFirewall::allowTrafficToRange(const IPAddress& addr, uint8_t weight,
```

Comment:
Split tunnel supports routes with network mask. That's why we rewrite default "allowTraffic".